docs/chart/delete-helm-chart.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Please merge this with other Markdown files into one README.md that will be placed in `chart/githubconnector/` directory.